samples/DispatcherSample/Startup.cs

```diff
@@ -39,18 +36,20 @@ public DefaultDispatcherDataSource ConfigureDispatcher()
             {
                 Addresses =
                 {
-                    new TemplateAddress("{controller=Home}/{action=Index}/{id?}", new { controller = "Home", action = "Index", }, "Home:Index()"),
-                    new TemplateAddress("{controller=Home}/{action=Index}/{id?}", new { controller = "Home", action = "About", }, "Home:About()"),
-                    new TemplateAddress("{controller=Home}/{action=Index}/{id?}", new { controller = "Admin", action = "Index", }, "Admin:Index()"),
-                    new TemplateAddress("{controller=Home}/{action=Index}/{id?}", new { controller = "Admin", action = "Users", }, "Admin:GetUsers()/Admin:EditUsers()"),
+                    new TemplateAddress("{id?}", new { controller = "Home", action = "Index", }, "Home:Index()"),
+                    new TemplateAddress("Home/About/{id?}", new { controller = "Home", action = "About", }, "Home:About()"),
+                    new TemplateAddress("Admin/Index/{id?}", new { controller = "Admin", action = "Index", }, "Admin:Index()"),
+                    new TemplateAddress("Admin/Users/{id?}", new { controller = "Admin", action = "Users", }, "Admin:GetUsers()/Admin:EditUsers()"),
                 },
                 Endpoints =
                 {
-                    new TemplateEndpoint("{controller=Home}/{action=Index}/{id?}", new { controller = "Home", action = "Index", }, Home_Index, "Home:Index()"),
-                    new TemplateEndpoint("{controller=Home}/{action=Index}/{id?}", new { controller = "Home", action = "About", }, Home_About, "Home:About()"),
-                    new TemplateEndpoint("{controller=Home}/{action=Index}/{id?}", new { controller = "Admin", action = "Index", }, Admin_Index, "Admin:Index()"),
-                    new TemplateEndpoint("{controller=Home}/{action=Index}/{id?}", new { controller = "Admin", action = "Users", }, "GET", Admin_GetUsers, "Admin:GetUsers()", new AuthorizationPolicyMetadata("Admin")),
-                    new TemplateEndpoint("{controller=Home}/{action=Index}/{id?}", new { controller = "Admin", action = "Users", }, "POST", Admin_EditUsers, "Admin:EditUsers()", new AuthorizationPolicyMetadata("Admin")),
+                    new TemplateEndpoint("{id?}", new { controller = "Home", action = "Index", }, Home_Index, "Home:Index()"),
```

Comment:
What are the defaults for in this model?

Reply:
These are the things that provide the ambient values. It doesn't have to be part of the endpoint class directly.

Reply:
Yea, it feels like this could be on another type of endpoint but I guess it's fine for now. Ideally we would tell people to look at metadata to determine what the controller is.